app.js

```diff
@@ -2,4 +2,44 @@
 
 // initialize state
 
-// set event listeners to update state and DOM
\ No newline at end of file
+// set event listeners to update state and DOM
+
+import { compareNumbers } from './utils.js';
+
+
+const submitButton = document.getElementById('submit-guess');
+const playersGuess = document.getElementById('players-guess');
+const textDisplay = document.getElementById('text-display');
+const displayNumberOfGuessesLeft = document.getElementById('display-guesses-left')
```

Comment:
make sure you're looking out for those eslint errors (should have a semicolon on the end here)